### PR TITLE
GEN-2432 - styles(BankIdDialog): new design for BankIdDialog

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Update _BankId Dialog_ styles to match  [new designs](https://www.figma.com/design/p1TEzLLVqJB9gKDNTDVGU4/Hedvig.com?node-id=472-12560&t=BlerQTS3vhkgYQqW-4)

Observe that I didn't add every single detail as there's some aspects I'd need to iron out first. With that said, I think this is a good start.

Screenshot 2024-07-24 at 14 57 10
Screenshot 2024-07-24 at 14 57 38
Screenshot 2024-07-24 at 14 57 58

**Note**: Text keys that can be removed after this gets merged:

LOGIN_BANKID_EXPLANATION
LOGIN_BANKID_SKIP
LOGIN_BANKID_AUTHENTICATION_STEPS_DESKTOP

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
* **#4533** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4533?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @guilhermespopolin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/BankIdDialog/BankIdDialog.css.ts [hunk 1]
```diff
@@ -1,39 +1,30 @@
...
-  justifyItems: 'center',
   width: 'min(24rem, 100%)',
   marginInline: 'auto',
 })
 
-export const qrOnAnotherDeviceFooter = style({
-  display: 'flex',
-  flexDirection: 'column',
-  alignItems: 'center',
-})
```
Comment (guilhermespopolin) [inline apps/store/src/components/BankIdDialog/BankIdDialog.css.ts:34]: This got replaced by `yStack`

File: apps/store/src/components/BankIdDialog/BankIdDialog.css.ts [hunk 2]
```diff
@@ -1,39 +1,30 @@
...
   width: 200,
   aspectRatio: '1 / 1',
 })
 
-export const iconWithText = style({
-  display: 'flex',
-  justifyContent: 'center',
-  alignItems: 'center',
-  gap: tokens.space.xs,
-})
```
Comment (guilhermespopolin) [inline apps/store/src/components/BankIdDialog/BankIdDialog.css.ts:21]: Got replaced by `yStack`

File: apps/store/src/components/BankIdDialog/BankIdDialog.stories.tsx [hunk 3]
```diff
@@ -31,7 +31,22 @@ const Template: StoryFn<BankIdContextValue> = (contextValue) => {
   )
 }
 
-export const Pending = {
+export const Sign = {
```
Comment (guilhermespopolin) [inline apps/store/src/components/BankIdDialog/BankIdDialog.stories.tsx:34]: Have added more stories so it comprehends all of the scenarios.

File: apps/store/src/components/BankIdDialog/BankIdDialog.css.ts (hunk 2 above)
Comment (alebedev) [reply apps/store/src/components/BankIdDialog/BankIdDialog.css.ts:21]: Nice!

**Review by alebedev — APPROVED** [review]
Great work!

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Jul 25, 7:28 AM EDT**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4533).

**Pull request merged**